PlayCover/Views/Sidebar Views/AppFolderView.swift

```diff
@@ -0,0 +1,268 @@
+                        }
+                    }
+                }
+                .frame(maxWidth: .infinity, maxHeight: .infinity)
+            }
+        }
+        .navigationTitle("sidebar.appLibrary")
+        .navigationSubtitle(folder.name)
+        .toolbar {
+            ToolbarItem(placement: .primaryAction) {
+                Button {
+                    if installVM.inProgress {
+                        Log.shared.error(PlayCoverError.waitInstallation)
+                    } else if downloadVM.inProgress {
+                        Log.shared.error(PlayCoverError.waitDownload)
+                    } else {
+                        selectFile()
+                    }
+                } label: {
+                    Image(systemName: "plus.circle")
+                        .help("playapp.add")
+                }
+            }
+            ToolbarItem(placement: .primaryAction) {
+                Spacer()
+            }
+            ToolbarItem(placement: .primaryAction) {
+                Button {
+                    showSettings.toggle()
+                } label: {
+                    Image(systemName: "gear")
+                }
+                .disabled(selected == nil)
+            }
+            ToolbarItem(placement: .primaryAction) {
+                Picker("Grid View Layout", selection: $isList) {
+                    Image(systemName: "square.grid.2x2")
+                        .tag(false)
+                    Image(systemName: "list.bullet")
+                        .tag(true)
+                }.pickerStyle(.segmented)
+            }
+            ToolbarItem(placement: .primaryAction) {
+                Button {
+                    addSheetApps.toggle()
+                } label: {
+                    Image(systemName: "pencil")
+                        .help("folder.button.edit")
+                }
+            }
+        }
+        .searchable(text: $searchString, placement: .toolbar)
+        .onChange(of: searchString, perform: { value in
+            appsVM.searchText = value
+            appsVM.fetchApps()
+        })
+        .onAppear {
+            appsVM.searchText = ""
+            appsVM.fetchApps()
+        }
+        .onChange(of: isList, perform: { value in
+            UserDefaults.standard.set(value, forKey: "AppLibraryView")
+        })
+        .sheet(isPresented: $showSettings) {
+            if let selected = selected {
+                AppSettingsView(viewModel: AppSettingsVM(app: selected))
+            }
+        }
+        .sheet(isPresented: $addSheetApps) {
+            VStack {
+                HStack {
+                    TextField(text: $folder.name,
+                              label: {Text("folder.textfield.name")})
+                        .frame(height: 40)
+                    Picker(selection: $folder.icon, label: Text("Icon")) {
+                        ForEach(appFolderVM.icons, id: \.self) { icon in
+                            Image(systemName: icon)
```

Comment:
Can the icons be made bigger? Same with the folder icons when creating a new folder.

Reply:
Problem seems that I cannot change size on a Picker, for the app List i can since is a List. Will try to change the picker to something else so i can change size